src/index.html

```diff
@@ -22,6 +22,428 @@
+          src="images/logo.png"
+          alt="Logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__element">
+            <a
+              href="#apple"
+              class="is-active nav__link"
+            >
+              apple
+            </a>
+          </li>
+
+          <li class="nav__element">
+            <a
+              class="nav__link"
+              href="#samsung"
+            >
+              samsung
+            </a>
+          </li>
+
+          <li class="nav__element">
+            <a
+              class="nav__link"
+              href="#smartphones"
+            >
+              smartphones
+            </a>
+          </li>
+
+          <li class="nav__element">
+            <a
+              class="nav__link"
+              href="#products"
+              data-qa="nav-hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+
+          <li class="nav__element">
+            <a
+              class="nav__link"
+              href="#gadgets"
+            >
+              gadgets
+            </a>
+          </li>
+
+          <li class="nav__element">
+            <a
+              class="nav__link"
+              href="#tablets"
+            >
+              tablets
+            </a>
+          </li>
+
+          <li class="nav__element">
+            <a
+              class="nav__link"
+              href="#photo"
+            >
+              photo
+            </a>
+          </li>
+
+          <li class="nav__element">
+            <a
+              class="nav__link"
+              href="#video"
+            >
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="catalog">
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            class="card__image"
+            src="./images/imac.jpeg"
+            alt="Product"
+          />
+
+          <div class="card__title">
+            <div class="card__title-text">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </div>
+            <div class="card__title-product-code">Product code: 195434</div>
+          </div>
+
+          <div class="card__stars stars">
+            <div class="stars__list">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <div class="stars__reviews">Reviews: 5</div>
+          </div>
+
+          <div class="card__price price">
+            <div class="price__text">Price:</div>
+            <div class="price__sum">$2,199</div>
+          </div>
+
+          <a
+            class="card__button button"
+            href="#"
+            data-qa="card-hover"
```

Comment:
The `data-qa` attribute for the 'Buy' link should be `data-qa="card-hover"` instead of `data-qa="hover"` to ensure consistency and accurate testing.